lms/djangoapps/discussion_api/api.py

```diff
@@ -487,7 +488,7 @@ def _check_editable_fields(cc_content, data, context):
     )
 
 
-def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context):
+def _do_extra_actions(api_content, cc_content, request_fields, actions_form, context, request):
```

Comment:
@wajeeha-khalid I am not familiar with this code but generally speaking the purpose would be to ensure that any callers of this function should know if they accidentally pass in an action that this function does not handle. This pattern is commonly used in logic which does some kind of dispatch.